src/lib.rs

```diff
@@ -225,6 +267,11 @@ impl Personnummer {
     pub fn serial(&self) -> u32 {
         self.serial
     }
+
+    /// The seperator between birthdate and serial.
+    pub fn seperator(&self) -> Seperator {
+        self.seperator
+    }
```

Comment:
When would you want only the separator as a consumer? Thinking it's not a part of the current spec and keeping the API to a minimum to reduce breakage and maintenance would be nice if possible. This would also allow us to not make `Separator` public.

Unless you are in need of the separator in your code?

Reply:
I guess that makes sense, let's keep it